config/validated_config_db_connector.py

```diff
@@ -0,0 +1,37 @@
+import jsonpatch
+from jsonpointer import JsonPointer
+
+from generic_config_updater.generic_updater import GenericUpdater, ConfigFormat
+
+def ValidatedConfigDBConnector(config_db_connector):
+    config_db_connector.set_entry = validated_set_entry
+    return config_db_connector
+
+def make_path_value_jsonpatch_compatible(table, key, value):
```

Comment:
>make_path_value_jsonpatch_compatible [](http://example.com/codeflow?start=4&length=36)

This will be a common function. Suggest add enough unit test cases.

Reply:
Added unit tests